src/DatabaseFunctions.js

```diff
@@ -1,6 +1,12 @@
 export function setUpDatabase() {
-    const defaultExerciseDatabase = [{value: 'Bench', text: 'Bench'}, {value: 'Deadlift', text: 'Deadlift'}, {value: 'Squat', text: 'Squat'}];
```

Comment:
You can remove the word Database from the variables.

Just move this file to src/Gateway/Database.js

Then name the variables `defaultExercises` `exercises` `defaultLogs` `logs`

Also why do you need this setUpDatabase function to create default entries?